app/presenters/summary/html_presenter.rb

```diff
@@ -21,11 +21,12 @@ class HtmlPresenter
         other_income_details
         housing_payments
         other_outgoings_details
+        properties
         savings
         premium_bonds
         national_savings_certificates
         investments
-        properties
```

Comment:
Did you mean to remove properties?

Reply:
I moved it as it's meant to display above savings